.github/actions/multi-approvers/tests/main.test.ts

```diff
@@ -62,4 +87,30 @@ test("#main", { concurrency: true }, async (suite) => {
       "Multi-approvers action failed: unexpected event [push].",
     );
   });
+
+  await suite.test(
```

Comment:
Please add one more test for the input: "123, 9,  456". I'd like to document (via a test) how we handle spaces in the input.